# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dia/gcc4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur dia-0.94.orig/objects/custom/shape_info.h dia-0.94/objects/custom/shape_info.h
--- dia-0.94.orig/objects/custom/shape_info.h	2004-08-16 10:56:15.000000000 +0300
+++ dia-0.94/objects/custom/shape_info.h	2005-10-31 17:09:46.000000000 +0200
@@ -166,7 +166,6 @@
 
 void shape_info_realise(ShapeInfo* info);
 void shape_info_print(ShapeInfo *info);
-void parse_path(ShapeInfo *info, const char *path_str, DiaSvgGraphicStyle *s);
 
 /*MC 11/03 handy g_new0 variant for struct with variable size */
 #define g_new0_ext(struct_type, ext_size)		\
diff -Nur dia-0.94.orig/plug-ins/python/pydia-object.h dia-0.94/plug-ins/python/pydia-object.h
--- dia-0.94.orig/plug-ins/python/pydia-object.h	2004-08-16 10:56:20.000000000 +0300
+++ dia-0.94/plug-ins/python/pydia-object.h	2005-10-31 17:19:51.000000000 +0200
@@ -22,7 +22,7 @@
 
 #include <Python.h>
 
-#include "object.h"
+#include "lib/object.h"
 
 typedef struct {
     PyObject_HEAD
